csfieldguide/templates/interactives/no-help.html

```diff
@@ -0,0 +1,19 @@
+{% extends interactive_mode_template %}
+
+{% load static %}
+
+{% block html %}
+    <div class="row interactive center-align">
```

Comment:
You won't need to specify a row and column here, a simple center aligned `div` will work, or using auto horizontal margins (see: https://getbootstrap.com/docs/4.1/utilities/spacing/#horizontal-centering).